src/Symfony/Component/VarDumper/VarDumper.php

```diff
@@ -37,13 +37,13 @@ class VarDumper
      */
     private static $handler;
 
-    public static function dump(mixed $var)
+    public static function dump(mixed $var, string $sectionName = null)
```

Comment:
`$label`
adding a new argument should be done in a BC way

Reply:
This is a static method, so I don't think we have a BC issue here

Reply:
I would call it `$name` (for argument name) or `$key` (for array key)
Like this.
```suggestion
    /**
     * @param string|null $name
     */
    public static function dump(mixed $var, /*string $name = null*/)
```

Reply:
I think I would go for `$label` which is clearer and more suitable to the context of "displaying an information".

I'll do the change for the BC promise, as I couldn't find any exception mentioning static methods, to add a new argument. :+1:

Reply:
it's a BC break, see https://3v4l.org/2cUek